app/views/admin/firms/show.html.erb

```diff
@@ -77,6 +77,11 @@
       <li><%= link_to adviser.name, admin_adviser_path(adviser) %>, added <%= adviser.created_at.to_s(:short) %></li>
     <% end %>
   </ul>
+
+  <div>
```

Comment:
Some people recommend against using divs/spans with no attributes. Do we need this wrapper? & if so, is there a more meaningful element we could use here?